### PR TITLE
Add: PS Script to generate .txt from .docx and .doc

### PR DESCRIPTION
**Pull request by wlawt**
This PR focuses on creating a Powershell (PS) script ([generate_txt.ps1](https://github.com/SheetJS/js-word/pull/114/files#diff-8ab1026ea798dd7df9973a95d7f29b5c)) that will open up all the `test_files` files and save them as plain text `.txt`

At the moment, currently supports `.doc .docx`. 

**To run this script:** 
1. Run `Set-ExecutionPolicy RemoteSigned` OR `Set-ExecutionPolicy -Scope Process -ExecutionPolicy Bypass` in PS Admin 7.0
2. Close and rerun PS Admin 7.0
3. Have the PS script in the root of the repo
4. Run `.\generate_txt.ps1 .\test_files\EXT_TYPE\FOLDER` (ex. `.\generate_txt.ps1 .\test_files\docx\apachepoi`)

**If you need to generate more plain text files:**
1. Run the script
2. If the script requires a password you'll have to uncomment [L27-29](https://github.com/SheetJS/js-word/pull/114/files#diff-8ab1026ea798dd7df9973a95d7f29b5cR27-R29) and comment [L26](https://github.com/SheetJS/js-word/pull/114/files#diff-8ab1026ea798dd7df9973a95d7f29b5cR26) and run the script
3. Finally, undo step 2

**Results that you should be seeing:**
- Word should be opening and closing (to grab and save the contents as `.txt`)
- There should be `.txt` files being generated in the same folder that you ran the script

**Other edge cases to consider:**
- The script will fail for documents that are broken (Word will prompt a notification to handle this)

### CODE REVIEW

**Comment by SheetJSDev** [comment]
Can you put the instructions for running the script somewhere in the repo (either as a section in the README.md or in a separate notes file)?

File: generate_txt.ps1 [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    $AbsPathI = Resolve-Path $AbsPath[$i] | Select-Object -ExpandProperty Path
+
+    if (Test-Path ($AbsPathI + ".txt") -PathType Leaf ) { continue main }
+
+    # $filename = Split-Path $AbsPath[$i] -leaf
+    # $join_fn = Join-Path -Path $Args[0] -ChildPath $filename
+    <# if (($join_fn | Out-String) -contains $Blacklist) { 
+        Write-Output "Skipping $join_fn"
+        continue main
+    } #>
```
Comment (garrettluu) [inline generate_txt.ps1:24]: Is the blacklist no longer being used?

File: generate_txt.ps1 [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    # Get-Process | Out-File -FilePath ".\$p\$filename.skip"
+    # continue main
+
+    Start-Process winword.exe
+    $Word = New-Object -ComObject Word.Application
+    try {
+        $Doc = $Word.Documents.Open($AbsPathI, $False, $True, $False, "WordJS", "WordJS")
+        $Doc.SaveAs(($AbsPathI+".txt"), 7, $False, "", $False, "", $False, $False, $False, $False, $False, $Encoding, $False, $False, $LineEnding)
+        $Doc.Close()
+    } catch {}
```
Comment (garrettluu) [inline generate_txt.ps1:37]: Should probably do something when you catch errors

File: generate_txt.ps1 (hunk 1 above)
Comment (wlawt) [reply generate_txt.ps1:24]: At the moment, it's currently not - but I think the more reasonable solution (in the future) is to make a blacklist, which is why I left it there (for now).

**Pull request merged**